actions/create_document.lua

```diff
@@ -1,27 +1,29 @@
 event: ["request_post_document"]
 priority: 1
 
-local helpers = require("helpers")
+local content = require("content")
 
 -- POST /
 local post_uuid = uuid.v4()
-local file = io.open("content/" .. post_uuid, "w")
+local file_uuid = uuid.v4()
+
 local params = {
-    title = request.body.title,
     type = request.body.type,
+    uuid = post_uuid,
+    attributes = {
+        title = request.body.title,
+        body = request.body.body,
+        created = request.body.created or "",
+        updated = request.body.updated or "",
+    }
 }
 
-local yaml_string = yaml.dump(params)
-local document_text = yaml_string .. "\n...\n" .. request.body.text
-local document_params = helpers.split_document(document_text, post_uuid)
-
-file:write(document_text)
-file:close()
+content.write_file (post_uuid, file_uuid, params, request.body.text)
 
 return {
     headers = {
         ["content-type"] = "application/json",
         ["X-Request-ID"] = post_uuid 
     },
-    body = render("document.json", { document = document_params }),
+    body = json.dump(params)
```

Comment:
might as well fix this now
https://github.com/foundpatterns/torchbear/issues/51